### PR TITLE
chore: fixed serverless deprecated warning

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:
- update serverless yml as there are some deprecated warnings.

### CODE REVIEW

**Comment by simboonlong** [comment]
suspect is an issue with `serverless-s3-remover` at `/issues/21`, which is a dep that storage service depends. i don't have a fix for u currently maybe u can leave a comment as a TODO to revisit again

- https://www.serverless.com/framework/docs/deprecations#cli-options-extensions-type-requirement

File: src/email/serverless.yml [hunk 1]
```diff
@@ -27,13 +28,13 @@ custom:
...
+    false: "true" # To negate the environment variable DISABLE_DOMAIN
+    true: "false" # if you know of a way to do conditionals please change this
   customDomain:
-    domainName: "${env:DOMAIN}" # Change this to your domain.
+    domainName: ${env:DOMAIN, "example.com"} # Change this to your domain.
     basePath: "email" # This will be prefixed to all routes
     stage: ${self:provider.stage}
-    certificateName: "${env:CERTIFICATE_DOMAIN}"
+    certificateName: ${env:CERTIFICATE_DOMAIN, "example.com"}
     createRoute53Record: true
```
Comment (simboonlong) [inline src/email/serverless.yml:38]: need `example.com`? or can omit

Comment (isaackps) [reply src/email/serverless.yml:38]: currently its having a default fall back of "example.com" if we dont have that we can either give another string url to fallback on or we can add it in the .env file. But currently the .env file doesn't have so i dont think it matters on local.

**Comment by simboonlong** [comment]
lgtm

**Review by simboonlong — APPROVED** [review]

**Pull request merged**